sdk/sdktypes/consts.go

```diff
@@ -2,5 +2,8 @@ package sdktypes
 
 import "go.autokitteh.dev/autokitteh/internal/kittehs"
 
-// REVIEW: we cannot define this in fixtures, due to circular dependencies
-var BuiltinSchedulerConnectionID = kittehs.Must1(ParseConnectionID("con_3kthcr0n000000000000000000"))
+// Note: we cannot define this in fixtures, due to circular dependencies
+var (
+	BuiltinSchedulerConnectionID = kittehs.Must1(ParseConnectionID("con_3kthcr0n000000000000000000"))
+	DefaultUser                  = NewUser("ak", map[string]string{"name": "Auto Kitteh", "email": "meow@autokitteh.com"})
```

Comment:
```suggestion
	DefaultUser                  = NewUser("ak", nil)
```

Reply:
those fields will be saved in DB as well so they cannot be nil
what's the name and email you want for the default user?

Reply:
changed to "a@k" and "dflt". Added hardcoded userID for the default user as well